docs/throttle/README.md

```diff
@@ -0,0 +1,33 @@
+# MinIO Server Throttling Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/)
+
+MinIO server allows two ways to throttle incoming requests
+
+- limiting the number of active requests allowed across the cluster
```

Comment:
```suggestion
- limit the number of active requests allowed.
```

Across the cluster? Isn't this per server?

Reply:
no this across cluster N/(number of hosts)